ietf/templates/nomcom/view_feedback_nominee.html

```diff
@@ -20,11 +20,26 @@ <h2>Feedback about {{ nominee }}</h2>
     </ul>
     <div class="tab-content">
         {% for ft in feedback_types %}
+            {% if is_chair and nomcom.group.state_id == 'active' and reclassify and nominee.feedback_set.all|length > 0 %}
+                <form class="form-inline" id="batch-action-form-{{ ft.slug }}" method="post">
+                {% csrf_token %}
+            {% endif %}
             <div role="tabpanel" class="tab-pane {% if forloop.first %} active{% endif %}"
                  id="{{ ft.slug }}">
                 {% for feedback in nominee.feedback_set.all %}
                     {% if feedback.type.slug == ft.slug %}
                         <dl class="row">
+                            {% if is_chair and nomcom.group.state_id == 'active' and reclassify and nominee.feedback_set.all|length > 0 %}
+                            <dt class="col-sm-2">
```

Comment:
An earlier iteration had the reclassification turned on full-time for the chair, and I felt that checkboxes would be less visually intrusive than a `Reclassify` button on each message. I eventually convinced myself that reclassification would be (or at least should be) infrequent enough that it should need to be explicitly enabled. Also for safety. So maybe now we want a `Reclassify` button at the bottom of each message, instead of a checkbox at the top.

Reply:
I later looked at a list that had more than one feedback item in the table and the checkbox was clearer than when I had only a single item, so maybe it's ok. Really the thing that concerned me was the lack of indication that the UI was in "reclassify mode". My suggestion of adding in the buttons could well add too much clutter - it was just a thought.